### PR TITLE
A submissions-per-queue management command

### PR DESCRIPTION
**Pull request by jibsheet**
Useful to have xqueue tell you in a tabular printout if you have a bunch
of pending submissions.  Also able to send data to New Relic Insights.

### CODE REVIEW

File: queue/management/commands/count_queued_submissions.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    def send_nr_metrics(self, queue_counts):
+        """
+        Send custom metrics to New Relic Insights
+        """
+        if not newrelic:  # pragma: no cover
+            raise CommandError("--newrelic cannot be used unless the newrelic library is installed")
+
+        for queue in queue_counts:
+            newrelic.agent.record_custom_metric(
+                    'Custom/XQueueLength/{}[{}]'.format(queue['queue_name'],
```
Comment (jibsheet) [inline queue/management/commands/count_queued_submissions.py]: @pwnage101 This is the metrics we were discussing.  I'll update when I have some data in insights from my testing which may make it clearer

Comment (pwnage101) [reply queue/management/commands/count_queued_submissions.py]: have an insights link for your sandbox i can peek at?

Comment (jibsheet) [reply queue/management/commands/count_queued_submissions.py]: What I've been doing is looking at the Data Explorer for Metrics in the edX-sandbox sub account of NR
https://insights.newrelic.com/accounts/1391042/dashboards/538614 should give you what I have

Comment (pwnage101) [reply queue/management/commands/count_queued_submissions.py]: Have you been able to extract the metrics via the insights query language?  I can't seem to find them for some reason.

**Comment by jibsheet** [comment]
@pwnage101 sorry, I rebased which squished your question.  The dashboard comes from the Metrics Explorer, not NRQL itself

File: script/max_pep8_violations [hunk 2]
```diff
@@ -4,7 +4,7 @@ DEFAULT_MAX=71
 pycodestyle . | tee /tmp/pep8-xqueue.log
 ERR=`grep -c ^ /tmp/pep8-xqueue.log `
 MAX=${1-$DEFAULT_MAX}
-if [ $ERR -g $MAX ]; then
+if [ $ERR -gt $MAX ]; then
```
Comment (coryleeio) [inline script/max_pep8_violations:7]: +1

**Review by coryleeio — APPROVED** [review]

**Pull request merged**